public/locales/fr-FR/grafana.json

```diff
@@ -46,28 +46,23 @@
     },
     "counts": {
       "alertRule__one": "",
-      "alertRule__many": "",
```

Comment:
not sure why these have been removed in french/spanish, but i think that's what's causing drone to fail 😬

Reply:
I noticed this (but didn't catch the drone failure). I think this is due to a disagreement of plural rules between i18next and crowdin?

This is... irritating. Will investigate.

Reply:
Ahh. The reason for this is that the English catalogue is the main reference. Any strings that exist in other languages but don't exist in English will be deleted.

As English doesn't have the `__many` plural, Crowdin will delete that translation key from all the other locales. It's is... problematic and will need to investigate the proper solution for it.

Semi related, because we don't have English __many plurals, this means that they're not able to be translated in Crowdin. The solution to both of these problems might be the same.